session.py

```diff
@@ -0,0 +1,671 @@
+            'submit' : 'Login',
+            'csrfmiddlewaretoken':csrftoken
+            }
+        self.loginResponse = self.s.post(self.loginURL,data = self.payload, headers=dict(Referer=self.loginURL))
+        return [csrftoken,sessionId]
+
+    #requests
+
+    def get(self, type, id = None):
+        """ Makes any get request
+            
+            args:
+                type ( str ): object of the request
+                id ( int, optional): id of the request
+
+            returns:
+                list: [response of the request, check(response)]
+        """
+        # create the URL by adding type and id
+        get_URL = self.base_URL + type + "/"
+        if id:
+            get_URL += str(id)
+
+        # make the request
+        response_get = self.s.get(get_URL, headers = self.headers)
+
+        return [response_get, mySession.check(response_get)]
+
+    def post_workout(self):
+        """ Makes a post request for a workout
+
+            returns:
+                list: [response of the request, check(response)]
+        """
+        # create the URL
+        post_workoutURL = self.base_URL + "workout/"
+
+        # create referer and headers
+        workout_post_headers = self.headers
+        post_workoutReferer = self.base_workoutReferer + "overview/"
+        workout_post_headers['Referer'] = post_workoutReferer
+        
+        # make the request
+        response_post_workout = self.s.post(post_workoutURL, headers= workout_post_headers)
+        return [response_post_workout, mySession.check(response_post_workout)]
+    
+    def post_training(self, workoutId, description, days):
+        """ Makes a post request for a training
+
+            args:
+                workoutId( int ): workout id where you want to add the training
+                description( str ): description of the training
+                days( list of int): the days of the week to make the training
+
+            returns:
+                list: [response of the request, check(response)]
+        """
+        # create the URL
+        post_trainingURL = self.base_URL + "day/"
+
+        # create referer and headers
+        training_post_headers = self.headers
+        post_trainingReferer = self.base_workoutReferer + str(workoutId) + "/view/"
+        training_post_headers['Referer'] = post_trainingReferer
+
+        # create the payload
+        payload_training = {
+            'training':workoutId,
+            'description':f'{description}',
+            'day':days
+            }
+
+        # make the request
+        response_post_training = self.s.post(post_trainingURL, json= payload_training ,headers = training_post_headers)
+        return [response_post_training, mySession.check(response_post_training)]
+    
+    def post_exercise(self, workoutId, exerciseday, exercises):
+        """ Makes a post request for an exercise
+
+            args:
+                workoutId( int ): workout id where you want to add the exercise
```

Comment:
I think this is the workout ID of the training in which you want to add the exercise.